lib/take_away.rb

```diff
@@ -0,0 +1,70 @@
+require_relative 'send_sms'
+
+class TakeAway
+  attr_reader :menu
+
+  def initialize
+    @menu = []
+    @basket = []
+    @total = 0
+    import_file
+  end
+
+  def read_menu
+    menu.each do |item|
+      puts "#{item[:index]}. #{item[:item]}: £#{item[:amount]}"
+    end
+  end
+
+  def order(item_number, quantity = 1)
+    dish = menu[item_number - 1]
+    @basket << { item_number: item_number, quantity: quantity }
+    "You have added #{quantity} x #{dish[:item]} to your basket."
+  end
+
```

Comment:
Maybe the name of the method could be changed? "order" could imply that it is the final order (which you have called "checkout" which is good), so is ambiguous. Maybe "add_to_basket" would be clearer.